### PR TITLE
feature/MAGICDRAW-659

### PR DESCRIPTION
**Pull request by w3e**

### CODE REVIEW

File: src/main/java/gov/nasa/jpl/mbee/mdk/MDKPlugin.java [hunk 1]
```diff
@@ -95,6 +98,9 @@ public void init() {
         loadExtensionJars(); // people can actually just create a new plugin and
 
         configureEnvironmentOptions();
+
+        SpecificationDialogManager.getManager().addConfigurator(Element.class, new SpecificationNodeAspectsConfigurator());
```
Comment (ivan-gomes) [inline src/main/java/gov/nasa/jpl/mbee/mdk/MDKPlugin.java]: Will this cause a new tab in the Specification window? If so, it needs to be removed until that feature is completed.

**Comment by w3e** [comment]
Turned off specification window add-in.

File: src/main/java/gov/nasa/jpl/mbee/mdk/systems_reasoner/actions/AspectSelectionAction.java [hunk 2]
```diff
@@ -51,13 +50,47 @@ public void actionPerformed(ActionEvent e) {
...
+        {
+            // Get selected element in single selection mode.
+        //    BaseElement selected = dlg.getSelectedElement();
+
+            // Get selected elements in multiple selection mode.
+            List<BaseElement> selected = dlg.getSelectedElements();
+
+            for(BaseElement be : selected){
+                System.out.println(be.getHumanName());
+                Stereotype aspectSt = Utils.getStereotype("aspect");
```
Comment (ivan-gomes) [inline src/main/java/gov/nasa/jpl/mbee/mdk/systems_reasoner/actions/AspectSelectionAction.java]: Stereotypes should be gotten by ID using StereotypesHelper. ID should be a static constant.

File: src/main/java/gov/nasa/jpl/mbee/mdk/systems_reasoner/SpecificationNodeAspectsConfigurator.java [hunk 3]
```diff
@@ -0,0 +1,157 @@
+/**
+ * Copyright (c) 2014 NoMagic, Inc. All Rights Reserved.
```
Comment (ivan-gomes) [inline src/main/java/gov/nasa/jpl/mbee/mdk/systems_reasoner/SpecificationNodeAspectsConfigurator.java]: Erroneous copyright.

File: src/main/java/gov/nasa/jpl/mbee/mdk/systems_reasoner/SRConfigurator.java [hunk 4]
```diff
@@ -168,8 +168,8 @@ public ActionsCategory handleSingleNode(ActionsCategory category, ActionsManager
             specializeStructureRecursiveAction = new SpecializeStructureAction(classifier, false, ID_RECURSIVE, true, false);
             createBSTAction = new SpecializeStructureAction(classifier, false, ID_RECURSIVE_INDIVIDUAL, true, true);
             createInstanceMenuAction = new CreateInstanceMenuAction(classifier);
-            aspectAction = new AspectAction(classifier);
-            copyAction = new CopyAction(element);
+          //  aspectAction = new AspectAction(classifier);
```
Comment (ivan-gomes) [inline src/main/java/gov/nasa/jpl/mbee/mdk/systems_reasoner/SRConfigurator.java]: Remove unused code.

File: src/main/java/gov/nasa/jpl/mbee/mdk/systems_reasoner/SRConfigurator.java [hunk 5]
```diff
@@ -143,7 +143,7 @@ else if (element instanceof InstanceSpecification) {
         validateAction = new ValidateAction(validatableElements);
         category.addAction(validateAction);
         if (!classifiers.isEmpty()) {
```
Comment (ivan-gomes) [inline src/main/java/gov/nasa/jpl/mbee/mdk/systems_reasoner/SRConfigurator.java]: Remove unused loop.

**Pull request merged**